### PR TITLE
improvement: allow overriding default config

### PR DESCRIPTION
**Pull request by davidcheung**
## Description

Please explain the changes you made here and link to any relevant issues.

### Checklist

- [ ] Validation tests are passing
- [ ] README.md has been updated after any changes to variables and outputs. See https://github.com/commitdev/terraform-aws-zero/#doc-generation

### CODE REVIEW

File: modules/user_auth/main.tf [hunk 1]
```diff
@@ -14,6 +14,128 @@ locals {
     }
   }
 
+  kratos_values_override = {
```
Comment (bmonkman) [inline modules/user_auth/main.tf:17]: So clean!

File: modules/user_auth/main.tf [hunk 2]
```diff
@@ -39,6 +161,15 @@ resource "null_resource" "external_secret_custom_resource" {
   depends_on = [kubernetes_namespace.user_auth]
 }
 
+module "kratos_config" {
+  source                     = "cloudposse/config/yaml"
```
Comment (bmonkman) [inline modules/user_auth/main.tf:165]: Nice!

**Review by bmonkman — APPROVED** [review]
Love it, great implementation!

File: modules/user_auth/main.tf (hunk 2 above)
Comment (davidcheung) [reply modules/user_auth/main.tf:165]: yeah they have some cool features like merging with remote configs as well

**Pull request merged**